### ISSUE TITLE
Clockwork Knight (US/JP/EU) does not load

### ISSUE THREAD

**Issue by blisstik** [devmiyax/yabause#552]
Using ORA v1.1 image from Retro Arena on an Odroid XU4. Bios will load then go into black screen.

/dev/shm/runcommand.log

```
Parameters: 
Executing: /opt/retropie/emulators/yabause/yabasanshiro -a -r 2 -b /home/pigaming/RetroPie/BIOS/saturn_bios.bin -i "/home/pigaming/RetroPie/roms/saturn/Clockwork Knight (US).cue"
[?25l[?1c
```

**Comment by natelawrence** [comment]
It may be of interest to @devmiyax that when using the Japanese compilation disc of Clockwork Knight 1 and 2 (クロックワーク　ナイト　〜ペパルーチョの福袋〜) Clockwork Knight **does** boot.

It seems to me that some library that many early Saturn games use to boot has an issue with Yaba Sanshiro which later games stopped using. 

I have also noticed in the past that whereas BUG! 1 cannot boot past the intro video, BUG TOO! will boot into game without issue.

**Comment by devmiyax** [comment]
zyoukan does not boot.
gekan can boot
fukubukuro can boot

**Comment by blisstik** [comment]
Still exists in v2.10

**Comment by devmiyax** [comment]
fixed at 1.8.0
